src/scripts/main.js

```diff
@@ -1,3 +1,17 @@
 'use strict';
 
 // write code here
+// Знайти елементи
+const spider = document.querySelector('.spider');
+const wall = document.querySelector('.wall');
+
+// Прорахувати сентральну позицію
+
+const leftPosition = (wall.offsetWidth - spider.offsetWidth) / 2;
+const topPositon = (wall.offsetHeight - spider.offsetHeight) / 2;
```

Comment:
There is a typo in the variable name `topPositon`. It should be `topPosition` to maintain consistency and avoid potential errors.